pfioh/store-options-handler.py

```diff
@@ -0,0 +1,322 @@
+"""

```

Comment:
I would recommend that so when looking at it from the perspective of one implementation you don't get mixed up.